### PR TITLE
need to trim off ce- from header keys.

### LINKED ISSUE #7: Extensions are not stripped of ce- header
stuff is wrapped in ce- over and over. Fix that.

### PR DESCRIPTION
**Pull request by n3wscott**
fixes: https://github.com/cloudevents/conformance/issues/7
Signed-off-by: Scott Nichols <nicholss@google.com>

### CODE REVIEW

File: pkg/http/http.go [hunk 1]
```diff
@@ -83,7 +83,7 @@ func RequestToEvent(req *http.Request) (*event.Event, error) {
 	event.Attributes.Extensions = make(map[string]string)
 	for k, _ := range req.Header {
 		if strings.HasPrefix(strings.ToLower(k), "ce-") {
-			event.Attributes.Extensions[k] = req.Header.Get(k)
+			event.Attributes.Extensions[k[len("ce-"):]] = req.Header.Get(k)
```
Comment (duglin) [inline pkg/http/http.go:86]: Did you want to make it into a constant?

Comment (duglin) [reply pkg/http/http.go:86]: what happens when the header's name is `ce-` ?  Seems like an error condition and it should ignore the header for the purposes of saving CE extensions.

Comment (n3wscott) [reply pkg/http/http.go:86]: I am trying to not over engineer this tool

Comment (n3wscott) [reply pkg/http/http.go:86]: There is not a ton of validation in this tool. I think it should be fairly barebones.

**Review by markpeek — APPROVED** [review]
LGTM

**Pull request merged**